src/ripple/app/main/Application.cpp

```diff
@@ -255,13 +255,13 @@ class ApplicationImp
         void operator() (Duration const& elapsed)
         {
             using namespace std::chrono;
-            auto const ms (ceil <std::chrono::milliseconds> (elapsed));
+            auto const ms (ceil <milliseconds> (elapsed));
 
             lastSample_ = ms;
 
-            if (ms.count() >= 10)
+            if (ms >= 10ms)
```

Comment:
Nit: We have a variable `ms` and a suffix `ms`, so we're seeing code like `ms >= 10ms`. The code is clear, however I it does seem a _little_ weird. Maybe rename the `ms` variable?